### PR TITLE
Added a quick, simple way to run the fieldtest alone

### PR DESCRIPTION
**Pull request by chamer81**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -64,5 +64,7 @@ behave:
 
 example-django:
   extends: .cookiecutter
-  script:
-  - tests/field/example-django.sh
+  variables:
+    FIELDTEST_CHECKS: bandit,flake8,pylint,safety,kubernetes
+    FIELDTEST_TESTS: py37,behave
+  script: tox -e fieldtest -- django
```
Comment (bittner) [inline .gitlab-ci.yml]: Hmmm, that looks okay. Actually, it makes me think about whether we should encapsulate the execution details within the fieldtest script or have them explicit, like here.

I feel, there must be a way to make this even simpler. Let's talk at the office... :speaking_head:

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -21,16 +21,27 @@ can test only against the versions you already have installed on your machine
 as follows, wait for the build servers to cover the missing pieces and fix
 identified issues with additional commits:
 
-.. code-block:: bash
+.. code-block:: console
 
     tox -e flake8,pylint,py36,py37
 
+We have field tests to generate and deploy an example project from your
+current working version.  In order to run the deployment, you need to have
```
[the remark this answers is not in the document]
Comment (chamer81) [reply CONTRIBUTING.rst]: OK, that's probably clearer.

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -21,16 +21,27 @@ can test only against the versions you already have installed on your machine
...
+access to the GitLab repository of your target generated project (such as
+`example django`_), and you need to generate a Personal Access Token on 
+GitLab. (Top-right user menu > Settings > Access Tokens)
+
+.. code-block:: console
+
+    export GITLAB_API_TOKEN=<your personal access token>
+    tox -e clean,fieldtest -- django
```
[the remark this answers is not in the document]
Comment (chamer81) [reply CONTRIBUTING.rst]: Yes, that's in the latest version.

File: .gitlab-ci.yml (hunk 1 above)
Comment (chamer81) [reply .gitlab-ci.yml]: OK, we have changed it back to calling the script directly, as we discussed.

**Pull request merged**